docs/develop/03_workshop/04_attester/01_account.md

```diff
@@ -39,7 +39,7 @@ A person can memorize it, and use it later to re-generate their keypairs and add
 To generate an account, one method from the Polkadot crypto utility, and one method from the KILT SDK is needed:
 
 - `mnemonicGenerate()` - Generates a 12-word mnemonic
-- `makeKeypairFromSeed(seed)` - takes a mnemonic as an input, and outputs a `Keypair` instance.
+- `makeKeypairFromSeed(seed)` - takes a mnemonic as an input, and outputs a [`KiltKeyringPair`](https://kiltprotocol.github.io/sdk-js/interfaces/_kiltprotocol_types.KiltKeyringPair.html) instance. A `Keypair` and a `KeyringKeypair` different in what information they expose. While a `Keypair` exposes both its public and private keys, a `KeyringPair` is managed by a keyring, which does not allow exposing private keys, but only supports signing arbitrary payloads.
```

Comment:
A `Keypair` and a `KeyringKeypair` **differ** in what information they expose.

Reply:
I fixed this piece based on Adel's feedback: https://github.com/KILTprotocol/docs/pull/233/commits/63321d3d96f679eec9bb7e55900bd1118303f0bf.